File: remove_items_in_list/remove_items_in_list.py
# def remove_items_in_list(origin_items: list, remove_items: list) -> list:
#     return list(filter(lambda item: item not in remove_items, origin_items))

def remove_items_in_list(origin_items: list, remove_items: list) -> list:
    result = []
    for item in origin_items:
        if item not in remove_items:
            result.append(item)
    return result


ori_items = [int(v) for v in input("items: ").split()]
rm_items = [int(v) for v in input("remove items: ").split()]

new_list = remove_items_in_list(ori_items, rm_items)
print(" ".join([str(v) for v in new_list]))

File: remove_items_in_list/remove_items_in_list.js
const readline = require('readline');

// function removeItemsInList(originItems, removeItems) {
//     return originItems.filter(item => !removeItems.includes(item));
// }

function removeItemsInList(originItems, removeItems) {
    const result = [];

    for (let i = 0; i < originItems.length; i++) {
        const item = originItems[i];
        if (!removeItems.includes(item)) {
            result.push(item);
        }
    }

    return result;
}

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

rl.question("Items: ", itemsInput => {
    rl.question("Remove items: ", removeInput => {
        const originItems = itemsInput.split(' ').map(Number);
        const removeItems = removeInput.split(' ').map(Number);

        const newList = removeItemsInList(originItems, removeItems);
        console.log(newList.join(' '));

        rl.close();
    });
});